lib/ramble/ramble/cmd/workspace.py

```diff
@@ -881,12 +908,28 @@ def workspace_edit(args):
         )
 
     config_file = ramble.workspace.config_file(ramble_ws)
+    applications_file = ramble.workspace.applications_file(ramble_ws)
+    modifiers_file = ramble.workspace.modifiers_file(ramble_ws)
+    software_file = ramble.workspace.software_file(ramble_ws)
```

Comment:
Are there potentially more such "include" files that we want to be able to edit? Would it make sense to have an `--all` option to open up all yaml files under the config directory, and then maybe another `--name` option to allow opening up specific ones (like `ramble workspace edit --name applications` to edit the applications.yaml file?)

Reply:
I'd love some feedback from Bob/Doug here. Yes, users could definitely add more files than just these and I think your two suggestions are good additions. It might also be nice for '--all' to open included files as well.

And as I'm using Ramble myself, I don't want to have to type out all those chars, which don't tab complete, to open just the 'applications.yaml' file. I like the convenience of having '-a' short flag for it. Do you think that having all these individual flags and options is making the interface too complex, or is it ok?

Reply:
In general I like and agree with all of the suggestion from both.  I think `--all` is good idea too (but definitely don't want that to be the default behavior) 

Slight preference for `-a` over `--type applications` personally, but am OK with either 

My overall take is that we should make the `top(n)` operations quick and easy, but we do not have to solve complex or edge cases and the user should `cd $RAMBLE_WORKSPACE; vim ...` for that

Reply:
Thanks Bob. The way it's written now, it keeps current behavior of opening ramble.yaml and templates by default. If they exist, it also opens applications, modifiers, and software. Adds flags for those 3 to support quick editing of them. Why don't I add in '--all' to cover cases where someone may use other filenames, and leave it there. Nothing changes in the workflow for existing users with just ramble.yaml, it works automatically if a user chooses to create app/mod/software files, and adds an easy option to open every yaml just in case someone wants it.